withdrawn-packages.txt

```diff
@@ -6,3 +72,4 @@ py3-google-cloud-pubsub-2.26.1-r0.apk
 py3-google-cloud-pubsub-2.27.0-r0.apk
 py3-google-cloud-pubsub-2.27.1-r0.apk
 py3-google-cloud-pubsub-2.27.2-r0.apk
+openjdk-24-24-r1.apk
```

Comment:
How did openjdk end up in the withdrawn-packages?

Reply:
nice catch. that's a failed conflict resolution. This file is annoying in that it is constantly added to and cleared after the removal has completed, so ripe for merge conflicts. IMO, this should be replaced with a directory where each PR can add its own file. But that's beside the point, let me fix this.